asciidoctor-editor-plugin/src/main/java-eclipse/de/jcup/asciidoctoreditor/preview/AsciiDoctorEditorBuildSupport.java

```diff
@@ -324,10 +325,14 @@ protected boolean isNeedingAHiddenEditorFile(File editorFileOrNull, File fileToC
 
     protected Pattern createRemoveAbsolutePathToTempFolderPattern() {
         Path tempFolder = getEditor().getWrapper().getTempFolder();
-        String absolutePathToTempFolder = tempFolder.toFile().getAbsolutePath();
-        String asciidocOutputAbsolutePath = absolutePathToTempFolder.replace('\\', '/');
-        asciidocOutputAbsolutePath += "/"; // relative path is without leading /
-        return Pattern.compile(asciidocOutputAbsolutePath);
+        //Convert to URI as asciidoc convert file path to URI in html document.
+        //So if the path contains a space or a special character it will be percent encoded
+        URI absolutePathToTempFolder = tempFolder.toFile().toURI();
+        String path = absolutePathToTempFolder.getRawPath();
+        if (path.startsWith("/")) {
```

Comment:
I'm on windows and there might be a system compatibility issue with this starting `/`. I have to remove it on windows, maybe not on Unix system, can you try to leave it and make a test ?

Reply:
I tried on Linux and there is a different path behavior between Linux and windows.
I have update the PR to take this difference into account.